### ISSUE TITLE
Add UV parameters to SphereShape

### ISSUE THREAD

**Issue by kuruk-mm** [decentraland/unity-renderer#242]
Continuous from #139 issue. Add UV to Sphere Shape.

**Comment by kuruk-mm** [comment]
The UVs from a BoxShape and PlaneShape can be predicted (8 values per face or 4 pairs of Vector2D, 48 values for the BoxShape and 16 for PlaneShape) but on SphereShape this works differently.

The number of vertices are determined here:
https://github.com/decentraland/explorer/blob/dc2f2ced31c59a5860ee9936bd6a4582a53fd150/unity-client/Assets/Scripts/MainScripts/DCL/Helpers/PrimitiveMeshBuilder.cs#L26

And when we do the maths in this code:
```
int nbLong = 24;
int nbLat = 16;
Vector3[] vertices = new Vector3[(nbLong + 1) * nbLat + 2];
```
We get a total of 402 vertices.

That value could change in the future if we change something about the number of vertices that we render on a circle (or maybe we can change depends on the quality settings on the future).

So I don't know exactly how is the best option to expose the number of UVs of a SphereShape to the user.

I'm right about this? Let me know :smile_cat:

**Comment by olavra** [comment]
Pending conversation with @nearnshaw

**Comment by olavra** [comment]
Propuesta de Brian: Añadir un transform de textura al componente de material

**Comment by olavra** [comment]
This will be resolved with the implementation of #1499
